companion/src/radiodata.h

```diff
@@ -1213,7 +1217,7 @@ class GeneralSettings {
     unsigned int backlightColor;
     CustomFunctionData customFn[CPN_MAX_CUSTOM_FUNCTIONS];
     char switchName[18][3+1];
-    unsigned int switchConfig[18];
+    unsigned int switchConfig[CPN_MAX_SWITCHES];
```

Comment:
Nice, I had forgotten about that FIXME until now.  Also, good move to `getBoardCapability()`, I think.  (Though I'm still not in love with the globals floating around.)